vpd-tool/include/tool_json_utility.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
+
+#include <fstream>
+
+namespace vpd
+{
+namespace jsonUtility
+{
+
+/**
+ * @brief API to parse respective JSON.
+ *
+ * Exception is thrown in case of JSON parse error.
+ *
+ * @param[in] i_pathToJson - Path to JSON.
+ * @return Parsed JSON.
+ * @throw std::runtime_error
+ */
+inline nlohmann::json getParsedJson(const std::string& i_pathToJson)
```

Comment:
A use case will be required to merge this utility function in the repo. I dont see any call in this PR.

Reply:
This API is used in https://github.com/ibm-openbmc/openpower-vpd-parser/pull/512